metricbeat/module/elasticsearch/enrich/_meta/data.json

```diff
@@ -1,38 +1,55 @@
 {
-    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "@timestamp": "2020-08-27T11:22:47.838Z",
+    "@metadata": {
+        "beat": "metricbeat",
+        "type": "_doc",
+        "version": "8.0.0"
+    },
+    "service": {
+        "address": "localhost:9200",
+        "type": "elasticsearch"
+    },
+    "event": {
+        "module": "elasticsearch",
+        "duration": 10042779,
+        "dataset": "elasticsearch.enrich"
+    },
     "elasticsearch": {
         "cluster": {
-            "id": "et6blfihSoytMUvkpYtEKQ",
-            "name": "docker-cluster"
+            "name": "docker-cluster",
+            "id": "giRKn5pZQ0afY0YrRiXdxQ"
+        },
+        "node": {
+            "id": "9kDCSqOSQRqY9Fwu8tQLSQ"
         },
         "enrich": {
+            "remote_requests": {
+                "current": 0,
+                "count": 0
+            },
             "executed_searches": {
-                "total": 1
```

Comment:
This would be a breaking change, no? I think we should keep this field the same as before to avoid that.

Reply:
Yes, I reverted this change.